digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/TrafficLightService.scala

```diff
@@ -34,4 +37,87 @@ class TrafficLightService(val roadLinkService: RoadLinkService) extends PointAss
       OracleTrafficLightDao.create(TrafficLightToBePersisted(asset.linkId, asset.lon, asset.lat, mValue, municipality, username), username)
     }
   }
+
+  override def getByBoundingBox(user: User, bounds: BoundingRectangle): Seq[PersistedAsset] = {
```

Comment:
It seems like there is a lot of duplicated code we can have a method on the base class that has a function as a parameter, and then reuse it.